libraries/abstractions/transport/CMakeLists.txt

```diff
@@ -8,7 +8,8 @@ afr_set_lib_metadata(VERSION "1.0.0")
 afr_set_lib_metadata(IS_VISIBLE "true")
 
 set(src_dir "${CMAKE_CURRENT_LIST_DIR}/secure_sockets")
-set(inc_dir "${CMAKE_CURRENT_LIST_DIR}/include")
+# The single source of truth for the transport_interface.h is in the coreMQTT repository.
+set(transport_interface_dir "${AFR_ROOT_DIR}/libraries/c_sdk_v4/standard/coreMQTT/source/portable")
```

Comment:
How would this work if only http is picked instead of MQTT in OCW?

Reply:
We have spoken offline.